portfolio-project-matching-app/pages/test_dao.js

```diff
@@ -0,0 +1,114 @@
+import { useEffect } from 'react'
+import {
+    createAssociation,
+    createDoc,
+    createNewLike,
+    createNewProjectDoc,
+    createNewProjectsTechnologiesDoc,
+    createNewProjectsUsersDoc,
+    createNewUserDoc,
+    deleteAssociation,
+    deleteDoc,
+    deleteLike,
+    deleteProjectsUsersDoc,
+    deleteUserDoc,
+    getProjectById,
+    updateDoc,
+    updateProject,
+} from '../backend/dao'
+import { Project } from '../models/Project'
+import { User } from '../models/User'
+
+const test_createProject = () => {
```

Comment:
We will certainly end up deleting this. I ran some very manual and very basic tests on the work I was doing.